Codeinwp/optimole-wp
# Add confirmation modal to Offloading

### All Submissions:

 

### Changes proposed in this Pull Request:
This adds a confirmation dialog to Offloading/Rollback as described in the issue.

**One question I have** is that the strings don't really align how the functionality works. If you cancel, we don't revert, we just stop the action but the string suggests we will revert the action, so would be good to confirm if we can change that to something else.

Closes https://github.com/Codeinwp/optimole-service/issues/1086.

### How to test the changes in this Pull Request:

1. Make sure the Modal appears similar to design.
2. The modal should appear when you click on Rollback/Offload, or Stop these process.
3. Make sure the process starts/stops as before with the use of the modal.

### Other information:
